src/index.html

```diff
@@ -4,9 +4,519 @@
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <div class="header__container">
+        <a href="/" class="header__logo">
+          <img
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="/" class="nav__link is-active">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="/" class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="/" class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="/"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="/" class="nav__link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="/" class="nav__link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="/" class="nav__link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="/" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="container">
+
+      <article
+        class="product"
+        data-qa="card"
+      >
+
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="product__image"
+        >
+
+        <h2 class="product__title">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product__code">
+          Product code: 195434
+        </p>
+
+        <div class="product__rating">
+
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+
+          </div>
+
+          <p class="product__rating_reviews">
+            Reviews: 5
+          </p>
+
+        </div>
+
+        <div class="product__price">
+
+          <p class="product__price_title">
+            Price:
+          </p>
+          <p class="product__price_price">
+            $2,199
+          </p>
+
+        </div>
+
+        <a
+          href="#"
+          class="product__button"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+
+      </article>
+      <article
+              class="product"
+      >
+
+        <img
+                src="./images/imac.jpeg"
+                alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+                class="product__image"
+        >
```

Comment:
fix indentation for attributes. check everywhere

Reply:
Not fixed